appium/options/android/common/adb/adb_options.py

```diff
@@ -0,0 +1,227 @@
+# Licensed to the Software Freedom Conservancy (SFC) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The SFC licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from datetime import timedelta
+from typing import Any, TypeVar
+
+from appium.options.common.supports_capabilities import SupportsCapabilities
+
+C = TypeVar("C", bound="SupportsCapabilities")
+
+
+class AdbOptionsDescriptor:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def __get__(self, obj: C, cls: type[C]) -> Any:
+        if self.name == 'ADB_EXEC_TIMEOUT':
+            value = getattr(obj, 'get_capability')(self.name)
+            return None if value is None else timedelta(milliseconds=value)
+        return getattr(obj, 'get_capability')(self.name)
+
+    def __set__(self, obj:C, value: C) -> C:
```

Comment:
`value` is an instance of `C`

Reply:
> `value` is an instance of `C`

I have fixed it. Also, I have used single quotes for representing strings.